### PR TITLE
Update cluster-api to v0.1.9

### LINKED ISSUE #439: update to cluster-api v0.1.9
/kind bug

**What steps did you take and what happened:**
[A clear and concise description of what the bug is.]

want to upgrade to 0.1.9 to see whether the bugs I found in cluster-api still exist or not , but find https://github.com/kubernetes-sigs/cluster-api-provider-openstack/tree/master/vendor/sigs.k8s.io/cluster-api
seems contains all code (test, hack etc) which is different to other folder (only pkg)
@hidekazuna can you guide me on how to upgrade using go mod for 'cluster-api'

**What did you expect to happen:**

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Cluster-api version:
- Minikube/KIND version:
- Kubernetes version: (use `kubectl version`):
- OS (e.g. from `/etc/os-release`):

### PR DESCRIPTION
**Pull request by hidekazuna**
**What this PR does / why we need it**:

This PR updates cluster-api from v0.1.4 to v0.1.9.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #439 

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by jichenjc** [comment]
@hidekazuna we need update 
cmd/clusterctl/examples/openstack/provider-component/cluster-api/manager_image_patch.yaml
 to v0.1.9 ... and check the location here 
https://github.com/kubernetes-sigs/cluster-api/issues/1258

File: cmd/clusterctl/examples/openstack/provider-component/cluster-api/manager_image_patch.yaml [hunk 1]
```diff
@@ -7,5 +7,5 @@ spec:
   template:
     spec:
       containers:
-      - image: gcr.io/k8s-cluster-api/cluster-api-controller:v0.1.4
+      - image: us.gcr.io/k8s-artifacts-prod/cluster-api/cluster-api-controller:v0.1.9
```
Comment (sbueringer) [inline cmd/clusterctl/examples/openstack/provider-component/cluster-api/manager_image_patch.yaml]: intended to swtich to us.gcr?

Comment (hidekazuna) [reply cmd/clusterctl/examples/openstack/provider-component/cluster-api/manager_image_patch.yaml]: @sbueringer I do not know the exact reason, see kubernetes-sigs/cluster-api#1258

Comment (sbueringer) [reply cmd/clusterctl/examples/openstack/provider-component/cluster-api/manager_image_patch.yaml]: Ah okay, good enough :)

**Comment by jichenjc** [comment]
root@jj-master:~/go/src/sigs.k8s.io/cluster-api-provider-openstack/cmd/clusterctl# kubectl describe pod controller-manager-0 -n system  --kubeconfig kubeconfig
Name:           controller-manager-0
Namespace:      system
........
    Image:         us.gcr.io/k8s-artifacts-prod/cluster-api/cluster-api-controller:v0.1.9

spaces
NAME                     STATUS   ROLES    AGE   VERSION
openstack-master-9r8mq   Ready    master   28m   v1.15.0
openstack-node-47qhn     Ready    node     24m   v1.15.0


/approve

**Pull request merged**